### ISSUE TITLE
FPS issue on different browsers

### ISSUE THREAD

**Issue by yemtelazer** [marklovers/kaplay#148]
## **Describe the bug**
Low FPS on different browsers

## **Version**
@latest

## **To Reproduce**
Have map which is not complex, just arround 20 tiles with colliders enabled.

In deckstop chrome and chromiums give 60ish FPS, but firefox / safari <30 FPS
In mobile chrome and chromiums give 60ish FPS, firefox 15-17 

## **Expected behavior**
Same performance across all browsers

## **Screenshots**
not taken

**Comment by niceEli** [comment]
This is an issue based on the computer and the browser engine itself. This is not an issue on KAPLAY, but instead is an issue with chromium, WebKit, and Gekko.

**Comment by yemtelazer** [comment]
not agree, same pc different browsers, different fps

also same phone different browsers different fps

Tested other webGL libraries they did not have any issues while I am having with kaplay / kaboom.

as a result I think there are somethings need to be refactored.

This project has a potential, but just because of this issue I need to switch to phaser I guess.

Hope you can fix, just trying to help.

Good luck

**Comment by lajbel** [comment]
It can also be an issue in browser configuration, please check this https://www.geeksforgeeks.org/how-to-enable-or-disable-hardware-acceleration-in-various-browsers, if problem persist, we can reopen

**Comment by yemtelazer** [comment]
My point is, I loaded my game in chrome and firefox lets say

in chrome i have 60 fps, firefox 25

And in exact same device, exact same browsers, exact same sessions

I opened a game which made with phaser for example.
Both browsers had no issue with fps

There is something clearly wrong

as i just said, i am just trying to help. I hope you can find the solution somehow.

good luckkk!!!

**Comment by rishavs** [comment]
I do not think that webgl performance between these mainstream browsers differ by this much (50%+). I also haven't observed this in other engines. Chrome is generally more performant, but never to this extent. 

@yemtelazer , maybe you can provide the smallest possible code where you are seeing this difference, for better debugging?

**Comment by mflerackers** [comment]
Have you tried to profile to see what takes up all that extra time each frame?

**Comment by lajbel** [comment]
updates?

**Comment by lajbel** [comment]
we are working actively on fix the issues of FPS and browsers compatibility, this will be eventually fixed, but for now, is not very specific.
